client/src/components/NoticeDetail.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from 'react'
+import {compose} from 'redux'
+import {connect} from 'react-redux'
+import {withDataProviders} from 'data-provider'
+import {noticeDetailProvider} from '../dataProviders/noticesDataProviders'
+import {noticeDetailSelector} from '../selectors'
+
+import type {Notice, State} from '../state'
+
+export type NoticeDetailProps = {
+  notice: Notice,
+  match: {
+    params: {
+      id: string,
+    },
+  },
+}
+
+const NoticeDetail = ({notice}: NoticeDetailProps) => {
+  return (
+    <div>
+      <div>{notice.title}</div>
+      <div>{notice.text}</div>
+      {notice.kandidati.map((n) => (Array.isArray(n) ? '' : <div key={n.id}>{n.customer}</div>))}
```

Comment:
I guess it's not a question for you but why is this sometimes array? And what should we do with it?

Reply:
in current API, when you request the whole list of notices, you won't get all of the candidates (you only need them for their count in the list view (I guess)) - instead you just get empty arrays - when you request detail of the notice you get candidate object instead of those